qa/tasks/vstart_runner.py

```diff
@@ -583,7 +583,7 @@ def _asok_path(self):
         log.info("I think my launching pid was {0}".format(self.fuse_daemon.subproc.pid))
         return path
 
-    def mount(self, mount_path=None, mount_fs_name=None, mount_options=[]):
+    def mount(self, mount_path=None, mount_fs_name=None, mountpoint=None, mount_options=[]):
```

Comment:
This argument is unused in the function. Should it override the usage of self.mountpoint?

Reply:
Yeah, you are right, and fixed it now.

Reply:
I think it's better to not set `self.mountpoint` here. fuse/kernel mount should not be doing redundant things.

Better would be:

`def mount(self, ..., **kwargs):`

so that it ignores any args it does not understand.